src/new-panel.scss

```diff
@@ -0,0 +1,86 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-new-panel;
+
+.#{$block} {
+
+  $fd-panel-header-height: 2.75rem;
+  $fd-panel-compact-header-height: 2.75rem;
+  $fd-panel-header-padding: 0.5rem;
+
+  &,
+  &__content,
+  &__title,
+  &__header,
+  &__expand {
+    @include fd-reset();
+  }
+
+  &__header {
+    height: $fd-panel-header-height;
+    min-height: $fd-panel-header-height;
+    padding-right: $fd-panel-header-padding;
+    background-color: var(--sapGroup_TitleBackground);
+    border-bottom: solid 0.0625rem var(--sapGroup_TitleBorderColor);
+    display: flex;
+    align-items: center;
+  }
+
+  &__title {
+    @include fd-typography(var(--sapFontHeader5Size));
+
+    flex: 1;
+  }
+
+  &__content {
+    padding: 0.625rem 1rem 1.375rem 1rem;
+    overflow: auto;
+
+    &[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
+  &__expand {
+    width: 2.75rem;
+    height: $fd-panel-header-height;
+    min-height: $fd-panel-header-height;
+
+    @include fd-flex-center();
+
+    .#{$fd-namespace}-button {
+      @include fd-icon('slim-arrow-right');
+
+      &.is-expanded {
+        @include fd-icon('slim-arrow-down');
+      }
+    }
```

Comment:
this is ok but the dependeny  (.#{$fd-namespace}-button) can be avoided by adding `fd-panel__expand-button` or just `fd-panel__button`. In this case you don't have to pass `sap-icon--slim-arrow-down` in `<button class="fd-button fd-button--transparent sap-icon--slim-arrow-down">`

Reply:
I can agree with you, there should be added some class to `fd-button`, I think there shouldn't be class collision.